.goreleaser.yaml

```diff
@@ -0,0 +1,47 @@
+before:
+  hooks:
+    - make dep
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.buildTime={{.Date}}
+    main: ./cmd/keepalived-exporter
+archives:
+  - name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
```

Comment:
I see nowadays repositories are using the real arch for the release. Do you think we should also go for it?
e.g. [node_exporter](https://github.com/prometheus/node_exporter/releases/tag/v1.5.0) || [prometheus](https://github.com/prometheus/prometheus/releases/tag/v2.42.0)

Reply:
Makes sense. Made changes.